src/ir/helper_types.go

```diff
@@ -20,6 +20,5 @@ type Class struct {
 }
 
 type Doc struct {
-	PhpDocComment string
-	PhpDoc        []phpdoc.CommentPart
+	phpdoc.Comment
 }
```

Comment:
Maybe use `phpdoc.Comment` instead of `ir.Doc` directly and remove the `ir.Doc` type?

Reply:
Done.